code/lib/cli/src/generators/configure.ts

```diff
@@ -114,20 +115,34 @@ export async function configurePreview(options: ConfigurePreviewOptions) {
     return;
   }
 
+  const prefix = [
+    isTypescript ? `import type { Preview } from '@storybook/${options.rendererId}'` : '',
```

Comment:
I wonder how this is going to work with community frameworks, it's better if the entire package comes as a parameter, rather than half of the import

Reply:
I'm not sure as well. Do we have an community framework example that I could test?
@JReinhold Do you know more about this?

Reply:
Fixed